tools/cell_census_builder/tests/test_builder.py

```diff
@@ -46,15 +44,21 @@ def setup_class(cls) -> None:
         process_initializer()
 
         # The tile extent needs to be smaller than the default (2048) to work with the fixture
-        CENSUS_X_LAYERS_PLATFORM_CONFIG["raw"]["tiledb"]["create"]["dims"]["soma_dim_0"]["tile"] = 2  # type: ignore
-        CENSUS_X_LAYERS_PLATFORM_CONFIG["raw"]["tiledb"]["create"]["dims"]["soma_dim_1"]["tile"] = 2  # type: ignore
+        CENSUS_X_LAYERS_PLATFORM_CONFIG["raw"]["tiledb"]["create"]["dims"]["soma_dim_0"]["tile"] = 2
+        CENSUS_X_LAYERS_PLATFORM_CONFIG["raw"]["tiledb"]["create"]["dims"]["soma_dim_1"]["tile"] = 2
 
     @classmethod
     def teardown_class(cls) -> None:
         cls.td.cleanup()
 
-    @pytest.fixture
-    def h5ad(self) -> anndata.AnnData:
+    @dataclass
+    class Organism:
+        name: str
+        organism_ontology_term_id: str
+
+    ORGANISMS = [Organism("homo_sapiens", "NCBITaxon:9606"), Organism("mus_musculus", "NCBITaxon:10090")]
+
+    def h5ad(self, organism: Organism) -> anndata.AnnData:
         X = np.random.randint(5, size=(4, 4))
         # The builder only supports sparse matrices
         X = csc_matrix(X)
```

Comment:
Use [scipy.sparse.random](https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.random.html#scipy.sparse.random) to directly create a random sparse matrix.  Will be much better as it will  have sparsity (this code is actually a sparse encoding of a dense matrix)

Reply:
that's a good lesson. since not significant for a 4x4 matrix, leaving as is w/a TODO for now.